src/main/java/com/yaphet/chapa/client/ChapaClient.java

```diff
@@ -17,4 +17,6 @@ public interface ChapaClient {
     String get(String url, String secreteKey) throws Throwable;
 
     int getStatusCode();
+
+    int getStatus();
```

Comment:
Should this return `Integer` instead of `int`
```suggestion
    Integer getStatus();
```